.github/workflows/ci.yml

```diff
@@ -41,10 +50,29 @@ jobs:
     - name: Install required python packages
       run: |
         python -m pip install --upgrade pip
-        pip install wheel numpy jax jaxlib
+        pip install wheel numpy jax jaxlib mpi4py
```

Comment:
Can skip jax and jaxlib here - that was for simsopt.

Reply:
Actually I guess I must have put jax there a long time ago for pyQSC, I forget why. I think it can be removed though.